themes/digital.gov/src/scss/new/guides/_guide-nav.scss

```diff
@@ -1,121 +1,85 @@
+      // Flex for the arrows
+      @include u-display("flex");
+      @include u-flex("align-center");
 
-          background-image: url("../uswds/img/angle-arrow-right-black.svg");
-          background-position: center bottom;
-          background-repeat: no-repeat;
-          background-size: 0.49rem;
-        }
+      &:hover {
+        @include u-text("underline");
       }
+    }
 
-      .usa-current {
-        @include u-bg("primary");
-        @include u-text("white");
-
-        &:hover {
-          @include u-text("white");
-          @include u-bg("primary-dark");
-        }
+    .usa-current {
+      @include u-bg("primary");
+      @include u-text("white");
+      @include u-text("semibold");
 
-        .arrow-right {
-          background-image: url("../uswds/img/angle-arrow-right-white.svg");
-        }
+      &:hover {
+        @include u-text("white");
+        @include u-bg("primary-dark");
       }
+    }
 
-      // Adding arrows to any li that has child elements.
-      li {
-        @include u-text("normal");
-        @include u-font("sans", "xs");
-
-        a {
-          .arrow-right {
-            @include u-margin-left("05");
-
-            background-size: 0.42rem;
-          }
-        }
-
-        li {
-          @include u-font("sans", "2xs");
-          li {
-            @include u-font("sans", "3xs");
-          }
-        }
-      }
+    .dg-guide__nav-list--current-item {
+      background-image: url("../uswds/img/angle-arrow-right-white.svg");
     }
 
-    // When any child item is selected, the top-most parent li gets the .current class (via javascript)
-    li.current {
+    // When any child item is selected, the top-most parent li gets the .dg-guide__nav-list--current class (via javascript)
+    li.dg-guide__nav-list--current {
       @include u-position("relative");
 
-      & > a {
+      a.usa-current {
         @include u-text("semibold");
-
-        &::after {
-          content: "";
-
-          @include u-border-left("05", "primary-darker", "solid");
-          @include u-radius(0);
-          @include u-position("absolute");
-          @include u-top(0);
-          @include u-bottom(0);
-          @include u-left(0);
-          @include u-z(100);
-        }
+        @include u-position("static");
       }
 
-      a {
-        span.arrow-right {
-          @include u-margin-left("105");
-
-          transform: rotate(90deg);
-        }
+      ul {
+        @include u-display("block");
       }
+    }
 
-      a.usa-current {
-        @include u-position("static");
+    // Adding arrows to any li that has child elements.
+    li {
+      @include u-text("normal");
+      @include u-font("sans", "xs");
+
+      .dg-guide__nav-list-arrow-right {
+        background-size: 0.42rem;
+        margin-left: units("05");
       }
 
-      ul {
-        @include u-display("block");
+      li {
+        @include u-font("sans", "2xs");
+        li {
+          @include u-font("sans", "3xs");
+        }
```

Comment:
Is this much nesting necessary? Could we get rid of these styles?

Reply:
We don't need `80-82`, but will need `78` for sublists, mostly for subsite migrations in the future that might need this styling.